### ISSUE TITLE
300ms delay ontouchstart

### ISSUE THREAD

**Issue by diego-cortez** [fatlinesofcode/ngDraggable#202]
I was testing on iPad with the current version of ngDraggable and the user needs to wait 300ms before start dragging, otherwise, the item is not dragging.

Is there a way to remove the 300ms delay for iOS webkit mobile browsers?

**Comment by francoisgrogor** [comment]
+1

@diego-cortez did you find a solution for this?

**Comment by diego-cortez** [comment]
@francoisgrogor  yeah, use ngTouch angular directive.

**Comment by francoisgrogor** [comment]
Thanks for response. Could you elaborate on that please? Currently ngDraggable has a configuration option to increase/decrease the timeout to ensure the native scrolling on touch devices is not effected.

Please elaborate on how using ngTouch fits into this?

**Comment by diego-cortez** [comment]
Well, I just added ngTouch directive to remove the delay on tap.. What is you current problem?, maybe I can help if I see what are you trying to do.

**Comment by francoisgrogor** [comment]
My issue is, that i feel the start of the drag on the touch device is a bit laggy. 

So if I attempt to touch and drag too quick, the window will just scroll. Essentially I just feel that the drag initial action required to start the drag on a touch screen is not reactive enough

**Comment by diego-cortez** [comment]
Yeah, I felt that, thats why I used ngTouch, I'm not sure but I remember that issue was fixed on webkit mobile browser, I think I tested on iPad.. I have an example if you want to test. please send me a mail to diego.cortez at tutanota.de
